stress_test_model_loan_book.R

```diff
@@ -236,11 +232,8 @@ pacta_loanbook_results_full <- read_pacta_results(
 
 pacta_loanbook_results_full <- pacta_loanbook_results_full %>%
   format_loanbook_st(
-    # FIXME: these inputs should not be assumed given, but rather than adding
-    # odd arguments to the function, refactoring should rather allow for bulk
-    # processing of all investor_names and portfolio_names
-    investor_name = investorname_loan_book,
-    portfolio_name = investorname_loan_book,
+    investor_name = investor_name_placeholder,
+    portfolio_name = investor_name_placeholder,
```

Comment:
It's possible we may need the pf name as a dedicated variable again soon, but that doesn't matter for now of course